addon/resolvers/classic/index.js

```diff
@@ -1,4 +1,4 @@
-/* globals requirejs, require */
+/* globals requirejs, require, loader */
```

Comment:
I think we should import require from require, then we'd get the one we want automatically

Reply:
Sorry @rwjblue I don't get it, can you please elaborate?

Reply:
I believe he wants to to `import require from 'require'`